l10n_co_geo/data/res_bank.xml

```diff
@@ -0,0 +1,864 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You convinced me that putting it in a separate module is probably better. I'll put it `l10n_co_partner_bank_data` that makes attribution clear, and separates concerns.